cassandane/tiny-tests/JMAPCalendars/calendarevent_query

```diff
@@ -206,6 +206,37 @@ sub test_calendarevent_query
         wantTotal => 3,
         wantIds => [$eventB1{id}],
         wantFastPath => JSON::false,
+    }, {
```

Comment:
While you are it, maybe also add a test for an invalid operator? Don't know if we already test this somewhere else.